### ISSUE TITLE
Tab order is not always obvious

### ISSUE THREAD

**Issue by regan-sarwas** [regan-sarwas/map-series-storytelling-template-js#13]
The tab order goes to the map first, then the description (document order).  This is appropriate when the description is on the right, but the order should be swapped when the description is on the left (settable in the layout settings and moved with styling, not DOM manipulation).
